File: 2_Kernel/JavaKernel/src/main/java/com/seagosoft/tws/data/AccountSummaryData.java
package com.seagosoft.tws.data;

public class AccountSummaryData {

    public int reqId;
    public String account;
    public String tag;
    public String value;
    public String currency;

    public AccountSummaryData(int reqId, String account, String tag, String value, String currency) {
        this.reqId = reqId;
        this.account = account;
        this.tag = tag;
        this.value = value;
        this.currency = currency;
    }
}

File: 3_OrderSystem/KotlinEWrapperBackend/src/main/kotlin/com/seagosoft/tws/wrapper/structs/AccountSummaryData.kt
package com.seagosoft.tws.wrapper.structs

class AccountSummaryData(var reqId: Int, var account: String, var tag: String, var value: String, var currency: String) {
    override fun toString(): String {
        return "AccountSummaryData(reqId=$reqId, account='$account', tag='$tag', value='$value', currency='$currency')"
    }
}